pybio/torch/training/simple.py

```diff
@@ -6,41 +9,32 @@
     trange = range
 
 import torch
+
+from pybio.core.transformations import apply_transformations_and_losses
+from pybio.spec.spec_types import ModelSpec
+
 from torch.utils.data import DataLoader
 from pybio.torch.transformations import apply_transformations
 
-
-# TODO config is just a stub object right now, adapt this to
-# the actual config object from pythonbioimageio
-def simple_training(config, n_iterations=500, batch_size=4, num_workers=2, out_file="./weights.pytorch"):
+def simple_training(model_spec: ModelSpec, n_iterations: int, batch_size: int, num_workers: int, out_file: Union[str, Path, IO[bytes]]) -> torch.nn.Module:
```

Comment:
example in model.yaml
```yaml
...
training:
 ...  
 source: pybio.torch.training.simple.simple_training
  required_kwargs: [model_spec]
  optional_kwargs: {n_iterations: 25, batch_size: 4, num_workers: 2, out_file: ./weights.pytorch}
```

Reply:
I just meant breaking the line between arguments:
```py
def simple_training(model_spec: ModelSpec, 
                                n_iterations: int,
                                ...)
```
instead of 
```py
def simple_training(model_spec: ModelSpec, .....):
```
The line is too long right now ;).
(We should set up black liniting at some point to take care of these things, but let's not get ahead of ourselves.)

Reply:
in the current commit it's blacked properly:
https://github.com/bioimage-io/pytorch-bioimage-io/blob/aafff158e7bd46c9c95e919b2fc99a8bcd64afcd/pybio/torch/training/simple.py#L20-L22

Reply:
ok, that's much better then the initial commit already